go/store/types/value_store.go

```diff
@@ -404,7 +418,10 @@ func (lvs *ValueStore) bufferChunk(ctx context.Context, v Value, c chunks.Chunk,
 				return err
 			}
 		}
-		return lvs.cs.Put(ctx, c)
+
+		return lvs.cs.Put(ctx, c, func(ctx context.Context, c chunks.Chunk) (hash.HashSet, error) {
+			return hash.NewHashSet(), nil
+		}) // Using lvs.getAddrs here makes a few store/types tests fail
```

Comment:
This was kinda fixed, still one test failing here https://github.com/dolthub/dolt/pull/5096/files#diff-7943a083af8107048b13b1758b33001082c0d206a8135df1b8a7e926e22dc27dL346

Reply:
we definitely need to fix this before we land the project, this is the primary write path for most things. We're better off skipping those tests than leaving a noop `GetAddrsCb`